src/postgres/src/test/regress/expected/yb_pg_foreign_data.out

```diff
@@ -1489,38 +1463,34 @@ HINT:  See https://github.com/YugaByte/yugabyte-db/issues/1129. Click '+' on the
 Server: s0
 FDW options: (delimiter ',', quote '"', "be quoted" 'value')
+Inherits: fd_pt1
+Child tables: ct3,
+              ft3
 
--- ^^^ originally extra:
--- Inherits: fd_pt1
--- Child tables: ct3,
---               ft3
---
 \d+ ct3
--- ^^^ originally returns:
---                                     Table "public.ct3"
---  Column |  Type   | Collation | Nullable | Default | Storage  | Stats target | Description 
--- --------+---------+-----------+----------+---------+----------+--------------+-------------
---  c1     | integer |           | not null |         | plain    |              | 
---  c2     | text    |           |          |         | extended |              | 
---  c3     | date    |           |          |         | plain    |              | 
--- Inherits: ft2
--- 
+                                    Table "public.ct3"
+ Column |  Type   | Collation | Nullable | Default | Storage  | Stats target | Description 
+--------+---------+-----------+----------+---------+----------+--------------+-------------
+ c1     | integer |           | not null |         | plain    |              | 
+ c2     | text    |           |          |         | extended |              | 
+ c3     | date    |           |          |         | plain    |              | 
+Inherits: ft2
+
 \d+ ft3
--- ^^^ originally returns:
---                                        Foreign table "public.ft3"
---  Column |  Type   | Collation | Nullable | Default | FDW options | Storage  | Stats target | Description 
--- --------+---------+-----------+----------+---------+-------------+----------+--------------+-------------
---  c1     | integer |           | not null |         |             | plain    |              | 
---  c2     | text    |           |          |         |             | extended |              | 
---  c3     | date    |           |          |         |             | plain    |              | 
--- Server: s0
--- Inherits: ft2
--- 
+                                       Foreign table "public.ft3"
+ Column |  Type   | Collation | Nullable | Default | FDW options | Storage  | Stats target | Description 
+--------+---------+-----------+----------+---------+-------------+----------+--------------+-------------
+ c1     | integer |           | not null |         |             | plain    |              | 
+ c2     | text    |           |          |         |             | extended |              | 
+ c3     | date    |           |          |         |             | plain    |              | 
+Server: s0
+Inherits: ft2
+
 -- add attributes recursively
 ALTER TABLE fd_pt1 ADD COLUMN c4 integer;
 ALTER TABLE fd_pt1 ADD COLUMN c5 integer DEFAULT 0;
 ALTER TABLE fd_pt1 ADD COLUMN c6 integer;
-ALTER TABLE fd_pt1 ADD COLUMN c7 integer NOT NULL;
+ALTER TABLE fd_pt1 ADD COLUMN c7 integer; -- TODO: should have NOT NULL; setting NOT NULL results in "Invalid argument: Invalid column number 4"
```

Comment:
Setting `NOT NULL` triggers the `invalid column number` error. I would assume this is related to the the `ft2` table inheriting from the `fd_pt1` being cached and Postgres trying to execute the `not null` clause on a `ft2.<column-index-4>`. However, because adding columns to the base table does not trigger the cache refresh of the inheriting table, hence the error.

This is just my assumption at this stage.